src/contexts/DrawerContext/DrawerContext.tsx

```diff
@@ -6,4 +6,6 @@ export const DrawerContext = createContext<DrawerContextType>({
   content: undefined,
   setContent: () => undefined,
   close: () => undefined,
+  isClosing: false,
+  finishClosing: () => undefined,
```

Comment:
Isn't better to call this 'onDraweClose' or 'finishDrawerClose' or something like that? WDYT?

Reply:
I believe that it will become redundant to have `*Drawer*` in every action and context. And to avoid conflicts when using actions from other hooks, we can still do this:

```ts
const { close: closeDrawer } = useContext(DrawerContext)
```